libs/libfolly/Makefile

```diff
@@ -0,0 +1,55 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libfolly
+PKG_VERSION:=2019.05.06.00
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/facebook/folly/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=59e88d792e4c917e2f37ac22c230af0056bce8aa7bacf2d9e94ceb177a7c1079
+PKG_BUILD_DIR:=$(BUILD_DIR)/folly-$(PKG_VERSION)
+
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+PKG_BUILD_PARALLEL:=1
+CMAKE_OPTIONS:= \
+	-DCMAKE_POSITION_INDEPENDENT_CODE=True \
+	-DCXX_STD=gnu++14 \
+	-DFOLLY_HAVE_UNALIGNED_ACCESS_EXITCODE=OFF \
+	-DFOLLY_HAVE_LINUX_VDSO_EXITCODE=OFF \
+	-DFOLLY_HAVE_WCHAR_SUPPORT_EXITCODE=OFF \
+	-DHAVE_VSNPRINTF_ERRORS_EXITCODE=OFF \
+	-DFOLLY_HAVE_XSI_STRERROR_R_EXITCODE=0 \
+	-DBUILD_SHARED_LIBS=ON
+EXTRA_CXXFLAGS:=-Wno-cpp -Wno-maybe-uninitialized -Wno-attributes -Wno-terminate
```

Comment:
Are these necessary? Compilation locally succeeded without these.

Reply:
You're right, this release no longer requires them.